front/lib/models/assistant/conversation.ts

```diff
@@ -367,11 +374,17 @@ Workspace.hasMany(AgentMessageFeedback, {
   onDelete: "RESTRICT",
 });
 AgentMessage.hasMany(AgentMessageFeedback, {
+  as: "feedbacks",
   onDelete: "RESTRICT",
 });
 UserModel.hasMany(AgentMessageFeedback, {
   onDelete: "SET NULL",
 });
+AgentMessageFeedback.belongsTo(UserModel);
+AgentMessageFeedback.belongsTo(AgentMessage, {
+  as: "agentMessage",
+  foreignKey: { name: "agentMessageId", allowNull: false },
```

Comment:
This foreign key should be also set on the gentMessage.hasMany(AgentMessageFeedback relation above no?